package.json

```diff
@@ -146,7 +146,7 @@
     "@types/sinon": "^10.0.2",
     "@types/sinon-test": "^2.4.2",
     "@types/trusted-types": "^1.0.6",
-    "@types/vscode-notebook-renderer": "^1.60.0",
+    "@types/vscode-notebook-renderer": "1.60.0",
```

Comment:
Why are you changing this? Please revert all changes in this file, none of this should be necessary

Reply:
When I run yarn it gets the newest version of this file (1.72), which gives a build error: 
<img width="1224" alt="Screenshot 2022-11-04 at 9 17 56 AM" src="https://user-images.githubusercontent.com/18150417/200024847-4653c79f-eae9-4a6d-b9aa-ac8d749833da.png">
so I specified the version.

Reply:
If you undo all the changes in this file you shouldn't see that behavior 

https://classic.yarnpkg.com/en/docs/cli/install#toc-yarn-install

```
If yarn.lock is present and is enough to satisfy all the dependencies listed in package.json, the exact versions recorded in yarn.lock are installed, and yarn.lock will be unchanged. Yarn will not check for newer versions.
```

This change is completely unrelated to your PR so if anything you should be doing something like this in a separate PR with an explanation of why it's needed

Reply:
You also need to revert the changes in yarn.lock when you do that.